xen-bugtool

```diff
@@ -89,9 +89,10 @@ else:
 #   fails with ENOSPC if the root filesystem is full
 if sys.version < "3.7":
     zipfile_del = zipfile.ZipFile.__del__  # type: ignore[attr-defined] # mypy,pyright
-    def exceptionless_del(*argl, **kwargs):
+
+    def exceptionless_del(*arg1, **kwargs):
         try:
-            zipfile_del(*argl, **kwargs)
+            zipfile_del(*arg1, **kwargs)
```

Comment:
`argl` isn't a mis-spelling.  It's normal python, like `kwargs`.

It took a while to figure out what had changed here, given that the fonts are unhelpful.

Reply:
Doh, yes of course! Fixed!